src/EnergyPlus/SimulationManager.cc

```diff
@@ -209,12 +209,8 @@ namespace SimulationManager {
              state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, "RunPeriod:CustomRange") > 0 || state.dataSysVars->FullAnnualRun);
         state.dataErrTracking->AskForConnectionsReport = false; // set to false until sizing is finished
 
-        OpenOutputFiles(state);
-
         state.init_state(state);
 
-        GetProjectData(state);
-        Psychrometrics::InitializePsychRoutines(state);
```

Comment:
Fundamental change is merging line 212, 216, and 217 into init_state. The rest is a result of this change.